src/python/pants/backend/go/goals/tailor_test.py

```diff
@@ -46,7 +52,13 @@ def rule_runner() -> RuleRunner:
             *link.rules(),
             QueryRule(PutativeTargets, [PutativeGoTargetsRequest, AllOwnedSources]),
         ],
-        target_types=[GoModTarget, GoBinaryTarget, GoPackageTarget],
+        target_types=[
+            GoModTarget,
+            GoBinaryTarget,
+            GoPackageTarget,
+            GoSdkTarget,
+            GoSdkPackageTarget,
+        ],
```

Comment:
As an aside: these `target_type` declarations are AFAIK only used to expose symbols to BUILD files. If the type is hidden anyway, it shouldn't be necessary to include it here.

Reply:
Good point. Will remove `GoSdkPackageTarget`. (`GoSdkTarget` is still necessary because it is used for a synthetic target and that symbol needs to be available since I get an error otherwise.)